pkg/service/backup_backend_s3.go

```diff
@@ -88,6 +95,21 @@ func (s *BackupBackendS3) FullBackupList() ([]model.BackupDetails, error) {
 	return result, err
 }
 
+// filterFolders returns backup folders that were created in the specified time range.
+func filterFolders(folders []types.CommonPrefix, from, to int64) ([]types.CommonPrefix, error) {
```

Comment:
We have method s.GetTime, that fetches time from S3 (and cache it)
Here you fetch time based on folder name.
I think, we should use the same approach in both places.

Reply:
I don't want to make this extra API call when pre-filtering.